crossterm_cursor/src/cursor/ansi_cursor.rs

```diff
@@ -130,7 +129,6 @@ mod tests {
 
     // TODO - Test is ingored, because it's stalled on Travis CI
     #[test]
-    #[ignore]
```

Comment:
These don't work on TCI, should be disabled. See the comment above (in the code).

Reply:
Yea I saw the comments, just wanted to try it again. Unfortunately...